src/locales/tr.json

```diff
@@ -1,8 +1,183 @@
+      "password": "Şifre",
+      "logInBtn": "Giriş Yap",
+      "forgotPass": "Şifremi unuttum",
+      "needAcc": "Bir hesaba mı ihtiyacın var?",
+      "signUp": "Hesap Oluştur",
+      "logInGoogle": "Google ile giriş yap",
+      "logInFacebook": "Facebook ile giriş yap",
+      "alertFailLogin": "Giriş yapılamadı.",
+      "alertLoginSuccess": "Hesap başarıyla oluşturuldu"
+    },
+    "signup": {
+      "signUp": "Hesap Oluştur",
+      "name": "İsim",
+      "surname": "Soyisim",
+      "email": "Email",
+      "password": "Şifre",
+      "passwordConfirm": "Şifre Doğrulama",
+      "getStarted": "Başla",
+      "haveAnAccount": "Bir hesabım var",
+      "logIn": "Giriş Yap",
+      "passNotMatch": "Şifreler Eşleştirilemedi",
+      "failCreateAcc": "Hesap oluşturulamadı"
+    },
+    "forgotPassword": {
+      "passReset": "Şifre yenileme",
+      "email": "Email",
+      "resetBtn": "Şifre yenileme",
+      "login": "Giriş Yap",
+      "needAccount": "Bir hesaba mı ihtiyacın var?",
+      "signup": "Hesap Oluştur",
+      "alertMessage": "Gelen kutunu kontrol et!",
+      "alertError": "Şifre yenilenemedi"
+    },
+    "faceLogin": {
+      "welcome": "Hoşgeldiniz"
+    },
+    "logout": {
+      "alertSuccess": "Başaryıla Çıkış Yapıldı",
+      "alertLogoutFail": "Çıkış Yapılamadı",
+      "signout": "Çıkış yap",
+      "updateProfile": "Profilini yenile",
+      "logOut": "Çıkış yap"
+    },
+    "updateLoginInfo": {
+      "passNotMatch": "Şifreler Eşleştirilemedi",
+      "failUpdate": "Profil yenilenemedi",
+      "updateProfile": "Profilini yenile",
+      "email": "Email",
+      "password": "Şifre",
+      "passConfirm": "Şifre Doğrulama",
+      "update": "Güncelle",
+      "cancel": "İptal"
+    }
+  },
+  "about": {
     "about": "Hakkımızda",
-    "recentPosts": "Güncel Paylaşımlar"
+    "ourTeam": "Ekibimiz",
+    "teamDiscription": "Bu projenin arkasında deneyimli bir lider tarafından yönetilen beş kişiden oluşan yardımsever bir ekip var, lütfen herhangi bir yardım veya geri bildirim için bizimle iletişime geçmekten çekinmeyin",
+
+    "aboutUs": [
+      {
+        "title": "İlkelerimiz",
+        "paragraph": "Birlikte daha iyi bir dünya yaratabiliriz. Herkes ihtiyacı olmayan eşyaları bağışlayarak dünyatı daha güzel bir yer haline getirmeye katkıda bulunabilir. Bağışınız her ne olursa olsun bir başkasının hayatına katkıda bulunabilirsiniz"
+      },
+      {
+        "title": "Sitenin Anafikri",
+        "paragraph": "Bağış yapmaya istekli kişiler ile bunlara ihtiyacı olan kişiler arasındaki bağlantıyı sağlamak ve bağışı en çok ihtiyacı olan kişilere ulaştırmak."
+      },
+      {
+        "title": "Hakkımızda",
+        "paragraph": "Bir grup genç insan, eşitsizliğim olmadığı daha iyi bir dünyanın hayalini kuruyor ve ihtiyacı olan insanlara ihtiyaç duyulan desteği sağlama çabalarını entegre etmeye çalışıyor. Kendimizi geliştirmek için her zaman yeni fikirler edinmeye çalışıyoruz."
+      }
+    ],
+    "team": [
```

Comment:
I meant the whole about object even this team one, it’ll be better if you open the English beside the turkish one and compare every object in both files